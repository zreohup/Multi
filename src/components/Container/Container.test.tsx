import React from 'react'
import { render } from '@/src/tests/test-utils'
import { Container } from './index'
import { Text } from 'react-native'

describe('Container', () => {
  it('renders correctly with children', () => {
    const { getByText } = render(
      <Container>
        <Text>Test Child</Text>
      </Container>,
    )
    expect(getByText('Test Child')).toBeTruthy()
  })

  it('applies the correct styles', () => {
    const { getByTestId } = render(
      <Container testID="container">
        <Text>Test Child</Text>
      </Container>,
    )
    const container = getByTestId('container')
    expect(container.props.style).toMatchSnapshot()
  })
})
