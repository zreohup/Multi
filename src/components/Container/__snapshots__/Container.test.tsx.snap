// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container applies the correct styles 1`] = `
{
  "backgroundColor": "#FFFFFF",
  "borderBottomLeftRadius": 7,
  "borderBottomRightRadius": 7,
  "borderTopLeftRadius": 7,
  "borderTopRightRadius": 7,
  "flexDirection": "column",
  "paddingBottom": 16,
  "paddingLeft": 16,
  "paddingRight": 16,
  "paddingTop": 16,
}
`;
